pinot-plugins/pinot-batch-ingestion/pinot-batch-ingestion-spark-3.2/pom.xml

```diff
@@ -36,9 +36,7 @@
   <properties>
     <pinot.root>${basedir}/../../..</pinot.root>
     <phase.prop>package</phase.prop>
-    <scala.major.version>2.12</scala.major.version>
```

Comment:
This module is still called spark 3.2 :p
I think we can either change it to `pinot-batch-ingestion-spark-3`?

Reply:
Yep. I discussed about that in https://github.com/apache/pinot/issues/11701 and its related issue. I'm ok with changing that name